_posts/minutes/2016-10-07-minutes.markdown

```diff
@@ -0,0 +1,101 @@
+[LISTEN TO THIS MEETING](https://fccdl.in/iPvoj2r1R)

```

Comment:
What other way would you suggest? It's an external link. Is just positioning or elevating it with an h tag? Moving it to somewhere else in the document?